src/main/java/seedu/address/model/stall/Stall.java

```diff
@@ -52,6 +52,10 @@ public Menu getMenu() {
         return menu;
     }
 
+    public boolean hasStallReview() {
```

Comment:
maybe Stall should have an overloaded constructor that can take in Reviews as arguments too